wpeverest/user-registration
# Feature - Form field icons in user registration

### All Submissions:






### Changes proposed in this Pull Request:



### How to test the changes in this Pull Request:

1. Navigate to User registration Form builder Form Settings > Extras Setting > Enable form field icon
2.  Now, See the User Registration form in the front end icon is appeared or not.
3.  Again, Navigate to the User Registration settings>Extras>Enable login field icon.
4.  Check whether the icon appeared in the login form or not.

### Dependent PR
https://github.com/wpeverest/user-registration-extras/pull/13
https://github.com/wpeverest/user-registration-woocommerce/pull/20


### Types of changes:




### Other information:




### Changelog entry

> Feature - Form field icons in user registration
